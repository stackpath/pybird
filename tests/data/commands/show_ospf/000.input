
0001 BIRD 2.0.4 ready.
1014-foo_igp:
 RFC1583 compatibility: disabled
 Stub router: No
 RT scheduler tick: 1
 Number of areas: 1
 Number of LSAs in DB:	2
 	Area: 0.0.0.0 (0) [BACKBONE]
 		Stub:	No
 		NSSA:	No
 		Transit:	No
 		Number of interfaces:	2
 		Number of neighbors:	1
 		Number of adjacent neighbors:	1
0000