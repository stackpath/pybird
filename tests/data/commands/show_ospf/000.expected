[
    {"area_count": 1, 
    "areas": 
    [
        {"adjacent_count": 1, 
        "neighbor_count": 1,
        "name": "0.0.0.0", 
        "interface_count": 2 
        }
    ], 
    "name": "foo_igp", 
    "lsa_count": 2
}
]